dashboard/cron.py

```diff
@@ -261,39 +262,52 @@ def update_with_bq_access(self):
             # query to retrieve all file access events for one course
             # There is no catch if this query fails, event_store.events needs to exist
 
-            final_bq_query = []
+            final_query = []
             for k, query_obj in settings.RESOURCE_ACCESS_CONFIG.items():
                 # concatenate the multi-line presentation of query into one single string
                 query = " ".join(query_obj['query'])
 
                 if (data_last_updated is not None):
                     # insert the start time parameter for query
-                    query += " and event_time > @data_last_updated"
+                    query += f" {query_obj['query_data_last_updated_condition']} "
```

Comment:
Is this something not needed for Postgres or were you filling something different in here. I'd be somewhat reluctant to remove this incase someones configuration wasn't updated and maybe be something like.

```
# is_bigquery is defined somewhere earlier to tell whether the LRS is bigquery or not. 
if query_obj['query_data_last_updated_condition']:
  query += f" {query_obj['query_data_last_updated_condition']} "
elif is_bigquery:
  query += " and event_time > @data_last_updated"
```

Reply:
Its because postgres uses a different param embed syntax 

postgres version looks like this `AND event_time > %(data_last_updated)s`

I can make the suggested change though. would